cmd/main.go

```diff
@@ -103,3 +112,43 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// Function for fetching nsx health status and feeding it to the prometheus metric.
+func getHealthStatus(lastSeenStatus *string) error {
+    status := "healthy"
+	if err := nsxClient.NSXChecker.CheckNSXHealth(); err != nil {
+		status := "unhealthy"
+	}
+	// Return without updating the version if it stayed the same since last time.
```

Comment:
Why not return the same status again?

Reply:
if err happend, set the status to "unhealthy"